java-runtime-builder-app/src/test/java/com/google/cloud/runtimes/builder/config/domain/BuildContextTest.java

```diff
@@ -87,14 +88,42 @@ public void testWriteDockerFilesWithExistingDockerignoreTerminalNewline() throws
     assertEquals(expectedDockerIgnore, readFile(getDockerIgnore()));
   }
 
-  @Test(expected = IllegalStateException.class)
-  public void testWriteDockerFilesWithExistingDockerfile() throws IOException {
+  @Test
+  public void testWriteDockerFilesWithExistingDockerfileAtRoot() throws IOException {
```

Comment:
One final thing...
The two tests look very similar. I think an improvement would be to create a parameterized test method `testExistingDockerfile(String dockerPath, String expectedErrorMessage)` and then just have a one-liner in each of tests below calling the utility method. WDYT?